.github/workflows/kotlin-test.yml

```
@@ -53,9 +53,25 @@ jobs:
       - name: Setup Android NDK
         run: sdkmanager "ndk;28.2.13676358"
       - uses: ./.github/shared/setup-rust
-      - name: Lint
-        working-directory: ./app-android
-        run: ./gradlew lint --quiet
-      - name: Test
-        working-directory: ./app-android
-        run: ./gradlew test
+        # FIXME
+#      - name: Lint
+#        working-directory: ./app-android
+#        run: ./gradlew lint --quiet -PtargetABI=x86_64
+#      - name: Test
+#        working-directory: ./app-android
+#        run: ./gradlew test -PtargetABI=x86_64
+#      - name: Android Test
+#        working-directory: ./app-android
+#        run: ./gradlew itest -PtargetABI=x86_64 --no-parallel --max-workers=1
+      - name: Enable KVM
+        run: |
+          echo 'KERNEL=="kvm", GROUP="kvm", MODE="0666", OPTIONS+="static_node=kvm"' | sudo tee /etc/udev/rules.d/99-kvm4all.rules
+          sudo udevadm control --reload-rules
+          sudo udevadm trigger --name-match=kvm
+      - name: Android test
+        uses: reactivecircus/android-emulator-runner@v2
+        with:
+          api-level: 29
+          arch: x86_64
+          working-directory: ./app-android
+          script: ./gradlew :app:connectedFdroidDebugAndroidTest -PtargetABI=x86_64
```

```suggestion
          script: ./gradlew itest -PtargetABI=x86_64
```
Do you only run it for the app on purpose?